index.html

```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <div class="card">
+          <p class="name">Marco Ciccale</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/mciccaleb" target="_blank">@mciccaleb</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/mciccale" target="_blank">@mciccale</a>
+          </p>
+          <p class="about">
+            I&apos;m a Software Engineer in Madrid that loves Web Development, AI,
+            Functional Programming, Web3 and many other fields. I really enjoy using
+            TypeScript as my go-to language. I use Next.js, React and TailwindCSS for
+            my Web projects. I enjoy publishing tools for the community!
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://mciccale.live" target="_blank" title="First Resource">Personal Website</a>
+              </li>
+              <li>
+                <a href="https://www.npmjs.com/~mciccale" target="_blank" title="Second Resource">NPM Profile</a>
+              </li>
+              <li>
+                <a href="www.linkedin.com/in/marco-ciccale-baztan" target="_blank" title="Third resource">LinkedIn</a>
```

Comment:
Hi, thanks for your contribution, the linkedin link you provided lacked `https`, so it was invalid. You can fix it by commit the suggestion on github, or change and commit it locally, and push again. After that, I'll review and merge your PR, thanks.
```suggestion
                <a href="https://www.linkedin.com/in/marco-ciccale-baztan" target="_blank" title="Third resource">LinkedIn</a>
```